From 7887126cf794a44a6fffbccb2587bceff2ff4698 Mon Sep 17 00:00:00 2001
From: Kaushlendra Kumar <kaushalendra.kumar@intel.com>
Date: Wed, 9 Sep 2020 11:47:24 +0530
Subject: [PATCH] Changes in start script for guest_pm_control call

guest-pm-control script is getting called twice
in start_civ script. The second call keeps it in a
continuous loop resulting in increase in CPU temp.

Tracked-On: OAM-92836
Signed-off-by: Kaushlendra Kumar <kaushalendra.kumar@intel.com>
---
 scripts/start_civ.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/scripts/start_civ.sh b/scripts/start_civ.sh
index dea3ee6..4cb3a70 100755
--- a/scripts/start_civ.sh
+++ b/scripts/start_civ.sh
@@ -480,7 +480,6 @@ function set_guest_pm() {
         local guest_pm_qmp_sock=$WORK_DIR/qmp-pm-sock
         $guest_pm_ctrl_daemon $guest_pm_qmp_sock &
         GUEST_PM_CTRL="-qmp unix:$guest_pm_qmp_sock,server,nowait -no-reboot"
-        $guest_pm_ctrl_daemon "$guest_pm_ctrl_pipe" &
     fi
 }
 
-- 
2.28.0

